peerconnection.go

```diff
@@ -1159,6 +1159,29 @@ func (pc *PeerConnection) CreateDataChannel(label string, options *DataChannelIn
 		params.ID = *options.ID
 	}
 
+	if options != nil {
+		// Ordered indicates if data is allowed to be delivered out of order. The
+		// default value of true, guarantees that data will be delivered in order.
+		if options.Ordered != nil {
+			params.Ordered = *options.Ordered
+		}
+
+		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #7)
+		if options.MaxPacketLifeTime != nil {
+			params.MaxPacketLifeTime = options.MaxPacketLifeTime
+		}
+
+		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #8)
+		if options.MaxRetransmits != nil {
+			params.MaxRetransmits = options.MaxRetransmits
+		}
+
+		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #9)
+		if options.Ordered != nil {
+			params.Ordered = *options.Ordered
+		}
+	}
+
 	// TODO: Re-enable validation of the parameters once they are implemented.
```

Comment:
Can you delete the comments below that don't apply anymore (since you started implementing things hooray!)

Reply:
I will leave comments for parameters that are not implemented yet. I would replace the comment section with something like this:
```
	// TODO: Enable validation of other parameters once they are implemented.
	// - Protocol
	// - Negotiated
	// - Priority:
	// - BufferedAmount
	//
	// See https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api for details
	//
	// Example
	/*
		if options != nil {
			// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #10)
			if options.Protocol != nil {
				channel.Protocol = *options.Protocol
			}

			// https://w3c.github.io/webrtc-pc/#peer-to-peer-da	ta-api (Step #12)
			if options.Negotiated != nil {
				channel.Negotiated = *options.Negotiated
			}

			// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #15)
			if options.Priority != nil {
				channel.Priority = *options.Priority
			}
		}

		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #11)
		if len(channel.Protocol) > 65535 {
			return nil, &rtcerr.TypeError{Err: ErrStringSizeLimit}
		}

		// https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api (Step #14)
		if channel.Negotiated && channel.ID == nil {
			return nil, &rtcerr.TypeError{Err: ErrNegotiatedWithoutID}
		}

		if pc.sctpTransport.State == SCTPTransportStateConnected &&
			*channel.ID >= *pc.sctpTransport.MaxChannels {
			return nil, &rtcerr.OperationError{Err: ErrMaxDataChannelID}
		}
	*/
```
I can push the change tomorrow.

Reply:
Pushed